qa/tasks/cephfs/test_subvolume.py

```diff
@@ -213,6 +213,28 @@ def test_subvolume_vxattr_removal_without_setting(self):
         # cleanup
         self.mount_a.run_shell(['rm', '-rf', 'group/subvol3'])
 
+
```

Comment:
I would recommend to remove this newline since in Python we reserve 2 sequential newlines for special purpose.